ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -34,6 +34,24 @@ SENSOR_ZTM_ZHM_CO2M_VOCM_PMDM_SPLM_ZAQIM_ZSPM_ZWVDM:
   uses:
   - illuminance_sensor
 
+  SENSOR_ZTM_ZHM_CO2M_VOCM_PMDM_SPLM_ZAQIM_ZSPM_ZOC:
```

Comment:
please fix tabbing here

Reply:
Could you provide more clarification? 
Should ZOC be added to SENSOR_ZTM_ZHM_CO2M_VOCM_PMDM_SPLM_ZAQIM_ZSPM_ZWVDM existing entity type?

Reply:
I just mean that the canonical type name is indented/has extra spacing before it

Reply:
Hi @shambergoldstein , thanks for getting back to me on that.